docs/.vuepress/config.js

```diff
@@ -383,7 +377,7 @@ module.exports = {
   title: '',
   port: 8080,
   description: 'The headless CMS developers love.',
-  base: '/documentation/',
```

Comment:
Remember we need to revert this before we merge it

Reply:
(Unless we merge in my fix PR)

Reply:
Thanks for the reminder, Derrick!

I've just [reverted it](https://github.com/strapi/documentation/pull/288/commits/377f82ace01ca9c5e13d0131ec41b0f510ffb7cc) and I suggest we don't change anything on the repo until deploy next Wednesday. We'll switch to a new infra in a few weeks anyway, so that will be the perfect moment to reconsider everything. Thank you! :-)